[BUGFIX] Fix PSR-7 request is missing in CLI context

sat-hub opened the pull request:
This fixes issue #54. I took the patch by @stigfaerch for 8.2.2 and applied it to the master branch.

sat-hub committed:
[BUGFIX] Fix PSR-7 request is missing in CLI context.

sat-hub committed:
Merge branch 'bugfix/8.2'

# Conflicts:
#	Classes/TypoLink/LinkModifier.php

sat-hub commented:
Correction: The patch is originally made by @TimWilms, see #56.

stigfaerch commented:
@sat-hub 
When comparing your patch with the on made by @TimWilms, there is many more changes in the one of Tims:
https://github.com/FriendsOfTYPO3/jumpurl/compare/8.2.2...TimWilms:jumpurl:8.2.2-fix
Any idea what that is all about?

sat-hub commented:
Of course - these are just additional whitespaces because he added an additional if-then, which increases the indentation of many lines.
I thought that this is quite unnecessary and made the diff much larger than it must be, so I changed the additional if to a boolean condition `$isFrontendRequest` and added it to the existing if, which results in only one additional line of code and one changed line of code - much shorter.
